requirements.txt

```diff
@@ -1,5 +1,5 @@
 requests[socks]>=2.18
-pydantic>=1.7.3,<2
+pydantic>=2,<3
```

Comment:
I think you want to add `typing_extensions` to the `requirements.txt` as well.

Reply:
Good catch. Although it's already required by pydantic and the package, and for py38 only. :smiley: